app/routes/tags/$tag.tsx

```diff
@@ -41,6 +41,36 @@ export async function fetchTag(tagName: string): Promise<TagType | never[]> {
   })
 }
 
+export function Tag({
+  name,
+  questions: tqs,
+  showCount,
+}: {
+  name: string
+  questions?: RelatedQuestions
+  showCount?: boolean
+}) {
+  const [questions, setQuestions] = useState(tqs)
+  const pageIds = questions?.map((q) => q.pageid).join(QuestionState.COLLAPSED)
+
+  useEffect(() => {
+    const fetcher = async () => {
```

Comment:
the obvious downside of this is that there will be loads of extra queries flying around

Reply:
if we have them cached in CF, it should be fine 🤞